Cinecon/JsonHelper.cs

```diff
@@ -68,17 +70,20 @@ private static void AddMenu()
         }
 
         private static void AddReservations()
-            => Reservations = JsonConvert.DeserializeObject<List<Reservation>>(File.ReadAllText("Assets/reservations.json"));
+            => ReservationData = JsonConvert.DeserializeObject<ReservationData>(File.ReadAllText("Assets/reservations.json"));
 
         private static void AddRooms()
             => Rooms = JsonConvert.DeserializeObject<List<Room>>(File.ReadAllText("Assets/rooms.json"));
 
+        private static void AddEmailData()
+            => EmailData = JsonConvert.DeserializeObject<EmailData>(File.ReadAllText("Assets/email.json"));
```

Comment:
hoe kom ik hier aan ''email.json'' bestand?

Reply:
In dit bestand staan gevoelige gegevens, vandaar dat het er niet bij staat. Ik zal de bestanden apart naar jullie sturen.